src/Form/Annotation/AnnotationBuilder.php

```diff
@@ -76,31 +77,31 @@ public function getFormSpecification($entity)
         $metadata     = $this->objectManager->getClassMetadata(is_object($entity) ? get_class($entity) : $entity);
         $inputFilter  = $formSpec['input_filter'];
 
-        $formElements = array(
-            'DoctrineModule\Form\Element\ObjectSelect',
-            'DoctrineModule\Form\Element\ObjectMultiCheckbox',
-            'DoctrineModule\Form\Element\ObjectRadio',
-        );
+        $formElements = [
+            Element\ObjectSelect::class,
```

Comment:
@Maaslos-UG 

Using the `class` keyword instead of string literals could be addressed in a separate PR, too.

Personally, I'd always suggest to break large PRs into a number of small PRs, especially if they address entirely separate issues.

I'm pretty sure that 75% of what is addressed in this PR could already have been merged into `master` if broken apart into separate PRs.

What do you think?